contribs/gmf/src/query/window.scss

```diff
@@ -11,8 +11,8 @@ div.gmf-displayquerywindow {
   button.collapse-button {
     pointer-events: all;
   }
-  .details {
-    display: none;
+  div.animation-container.next {
```

Comment:
`div.animation-container` or `div.animation-container {&.ext, &.previous {...}}`
Because if you click on "previous" results disappears

Reply:
OK, thaks :-)